### PR TITLE
Support RSS 0.9

### PR DESCRIPTION
**Pull request by rukandax**
Add support for RSS 0.9

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -270,6 +276,8 @@ Parser.parseString = function(xml, options, callback) {
       return parseRSS2(result, options, callback);
     } else if (result['rdf:RDF']) {
       return parseRSS1(result, options, callback);
+    } else if (result.rss && result.rss.$.version && result.rss.$.version.indexOf('9') === 2) {
```
Comment (bobby-brennan) [inline index.js]: Just to make this a little more robust, can you change `.indexOf('9') === 2` to `.match(/0\.9/)`?

**Comment by bobby-brennan** [comment]
Looks great aside from the one comment. Thanks!

I'll be sure to add this to v3 as well.

File: index.js (hunk 1 above)
Comment (rukandax) [reply index.js]: sure

**Comment by bobby-brennan** [comment]
Published as v2.12.0

Thanks for the contribution!

**Pull request merged**